tests/bot.ts

```diff
@@ -8,11 +8,9 @@ import {
   INVALID_SESSION,
   ev,
 } from "../mod.ts";
-import * as dotenv from "https://deno.land/x/denoenv/mod.ts";
```

Comment:
Deno won't automatically pull vars from .env files into `Deno.env`, so this module is still needed. This can simply import the Load file:
```diff
- import * as dotenv from "https://deno.land/x/denoenv/mod.ts";
+ import "https://deno.land/x/dotenv@v1.0.1/load.ts";
```